src/app/(admin)/admin/components/Modal.tsx

```diff
@@ -1,121 +1,162 @@
-import { useState } from "react";
+// "use client";
```

Comment:
why?

Reply:
For now, still undecided about whether to add a pop up feature or not